targets/targets.json

```diff
@@ -6334,6 +6334,11 @@
         "release_versions": ["5"],
         "device_name": "nRF52832_xxAA"
     },
+    "MTB_ACONNO_ACN52832": {
```

Comment:
@ithinuel : Minor nit. could you please move this a little down below where 2 of the other MTB targets are defined based on this MCU. Just nagging to keep all the MTB targets in one place in the targets.json ..

Reply:
fixed in c0ca419